src/telemetry/telemetryService.ts

```diff
@@ -253,22 +267,24 @@ export default class TelemetryController {
     dataService: DataServiceType,
     connectionType: ConnectionTypes
   ): void {
-    dataService.instance({}, async (error: any, data: any) => {
```

Comment:
Since we are doing a lot of things depending on the instance results here - maybe we can promisify this and try catch it. Then if we do end up doing something that breaks it's handled well - maybe we also want to log that error to telemetry.